### ISSUE TITLE
Using animated marker with realtime GPS data

### ISSUE THREAD

**Issue by m-menon** [openplans/Leaflet.AnimatedMarker#40]
hi, thanks a lot for the fantastic plugin - looks cool. Had a small question as I am just getting started the leaflet - I have a situation where GPS data is received periodically by the server. How can I use your animated marker plugin to smoothly move the marker to the new gps location rather than just the current 'abrupt' jump method I am using. I guess I will have to do something like below?

```
var line = L.polyline([[Current Lat Lon],[New Lat lon]]),
    animatedMarker = L.animatedMarker(line.getLatLngs());

map.addLayer(animatedMarker);
```

Do I have to keep doing map.addLayer for every new gps position that I receiver? 

Thanks,
M&M

**Comment by harithFED** [comment]
you find a way on this?

**Comment by atogle** [comment]
No, `addLayer` simply adds the layer to the map, not points to the line. You would need to add points to the polyline layer instead. I don't know if this would work automatically but seem like it would be possible.

**Comment by harithFED** [comment]
okay2. im using Ivan plugin xD
